src/main/java/net/sf/mzmine/modules/peaklistmethods/peakpicking/deconvolution/DeconvolutionTask.java

```diff
@@ -59,6 +61,11 @@
     // User parameters
     private final ParameterSet parameters;
 
+	private RSessionWrapper rSession;
+	private String errorMsg;
+	private boolean userCanceled;

```

Comment:
Instead of this new userCanceled variable you can simply use the isCanceled() method